frontend/src/components/common/form/ParentSelect.test.tsx

```diff
@@ -58,15 +58,15 @@ it('changes to corresponding child value on click', async () => {
     </Formik>,
   );
   const test = container.querySelector('input[name="test"]');
-  await wait(() => {
+  await waitFor(() => {
```

Comment:
**wait** has been deprecated in the latest version of react-testing-library. Use **waitFor** instead